core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java

```diff
@@ -168,6 +175,23 @@
         return composite;
     }
 
+    /**
+     * Returns a system property composed of environment variables and system properties.
+     *
+     * <p>Properties present in system properties will
+     * take precedence over those in environment variables,
+     * similarly to spring boot behaviour.
+     */
+    @SuppressWarnings({"rawtypes", "unchecked"})
+    private PropertySource<?> loadEnvironmentAndSystemProperties() {
+        CompositePropertySource environmentAndSystemProperties = new CompositePropertySource("environmentAndSystemProperties");
```

Comment:
Pleas run the style guide to make sure you're up to speed.

Reply:
Done, I added the static modifier